stac_model/input.py

```diff
@@ -97,11 +99,11 @@ class ModelBand(MLMBaseModel):
         )
     )
     # similar to 'ProcessingExpression', but they can be omitted here
-    format: Annotated[Optional[str], OmitIfNone] = Field(
+    format: Annotated[str | None, OmitIfNone] = Field(
         default=None,
         description="",
     )
-    expression: Annotated[Optional[Any], OmitIfNone] = Field(
+    expression: Annotated[Any | None, OmitIfNone] = Field(
```

Comment:
If keeping `Optional` does not cause linting issues, I suggest keeping them as is. 
Somewhat inconsistent otherwise because `Optional` is used in all other places using `Annotated`.

Reply:
`make format` prefers replacing Optional with python built-ins, probably because it is more succinct and makes imports cleaner. ruff format also prefers not using List from typing, etc. I'd prefer to just follow ruff's convention since it makes cleaning up code to adher eto a standard style with make format way easier.

Reply:
switched to Optional. make format doesn't seem to change this anymore not sure why.